mesonbuild/linkers.py

```diff
@@ -927,6 +927,12 @@ def export_dynamic_args(self, env: 'Environment') -> T.List[str]:
     def import_library_args(self, implibname: str) -> T.List[str]:
         return ['--symdefs=' + implibname]
 
+class QualcommLLVMDynamicLinker(LLVMDynamicLinker):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        # ARM Linker from Snapdragon LLVM ARM Compiler
+        self.id = 'ld.qcld'
```

Comment:
Here it is. Since `ld.lld` is far more popular than `ld.qcld`, I prefer not to disturb the default behavior of `LLVMDyanamicLinker`.